packages/compiler-dom/src/htmlNesting.ts

```diff
@@ -63,7 +63,7 @@ const onlyValidChildren: Record<string, Set<string>> = {
     'template',
   ]),
   optgroup: new Set(['option']),
```

Comment:
Per [spec](https://html.spec.whatwg.org/dev/dom.html#optgroup-element-inner-content-elements-2), `div` is also allowed.